skype/skype-ps/skype/Remove-CsCallQueue.md

```diff
@@ -0,0 +1,115 @@
+---
+external help file: Remove-CsCallQueue.xml
+applicable: Skype for Business Online
+title: Remove-CsCallQueue
+schema: 2.0.0
+---
+
+# Remove-CsCallQueue
+
+## SYNOPSIS
+The Remove-CsCallQueue cmdlet deletes an existing Call Queue.
+
+## SYNTAX
+
+```
+Remove-CsCallQueue -Identity <System.Guid> [-AsJob] [-BypassDualWrite <System.Boolean>] [-Confirm]
+[-DomainController <Object>] [-Force] [-Tenant <System.Guid>] [-WhatIf] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The Remove-CsCallQueue cmdlet deletes an existing Call Queue specified by the Identity parameter.
```

Comment:
May be mention that Remove fails if there are any ApplicationInstances associated with the CallQueue?